conn.go

```diff
@@ -495,16 +498,16 @@ func (conn *Conn) send(ctx context.Context, msg *Message, ch chan *Call) *Call {
 	if ctx == nil {
 		panic("nil context")
 	}
+	if ch == nil {
+		ch = make(chan *Call, 5)
```

Comment:
Why 5? Typically buffered channels just need one slot unless you have reason to expect multiple messages.

Reply:
This is the capacity the code was already using: I just moved it out of the if block below.  I agree that it is weird, and the other copy of this code was using a capacity of 1, so I've reduced it here too.